<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainProfilerWidget</class>
 <widget class="QWidget" name="MainProfilerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Visualizer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>21</y>
        <width>1170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="statusBar">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load .csv file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveCsvButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save current as .csv</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save current profile as an image</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="legendButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Display legend</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mergeButton">
           <property name="text">
            <string>Merge controls</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="helpButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Help</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
